test/test_faiss_and_milvus.py

```diff
@@ -318,7 +324,10 @@ def test_faiss_cosine_similarity(tmp_path):
     for doc in query_results:
         result_emb = doc.embedding
         original_emb = np.array([indexed_docs[doc.content]], dtype="float32")
-        faiss.normalize_L2(original_emb)
+        if type(document_store_cosine)==FAISSDocumentStore:
+            document_store_cosine.normalize_embedding(original_emb)
+        else:
+            document_store_cosine.normalize_embedding(original_emb[0])
```

Comment:
Thank you for asking :-) np.linalg.norm(emb) was several times slower than np.sqrt(emb.dot(emb)) in my testing. A pity that I can't find exact timings comparison anymore. Maybe not a big deal, but if it can be changed back to np.sqrt(emb.dot(emb)) while still passing the tests, I'd use it.

Reply:
But if the intention was to use np.linalg.norm for the sake of unification, I'm fine with that, too.

Reply:
ok, I rolled back to plain numpy operations so that we get the performance boost..